src/DependencyInjection/SonataDoctrineORMAdminExtension.php

```diff
@@ -59,5 +60,21 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->getDefinition('sonata.admin.builder.orm_show')
             ->replaceArgument(1, $config['templates']['types']['show']);
+
+        // NEXT_MAJOR: remove this block.
+        $deprecatedEmptyFilterDefinition = $container->getDefinition('Sonata\DoctrineORMAdminBundle\Filter\EmptyFilter');
+        if (method_exists(BaseNode::class, 'getDeprecation')) {
```

Comment:
this is borrowed from https://github.com/doctrine/DoctrineBundle/blob/2.2.x/DependencyInjection/Configuration.php#L735